### PR TITLE
Resolves #2454 : DataTables in admin dashboard are made responsive by js

### LINKED ISSUE #2454: Make DataTables in admin dashboard responsive
DataTables are used in admin dashboard under most of the tabs and they are overflowing on small screens.

Screenshots : 

localhost-5000-admin-events- iphone 6 plus 2

localhost-5000-admin-events- iphone 6 plus 1

### PR DESCRIPTION
**Pull request by sainiak009**
I have defined a callback function in javascript which will adjust and keep in sync DataTable's width according to window size. Tried to solve it with CSS only but didn't worked.

### CODE REVIEW

**Comment by sainiak009** [comment]
Screenshots : 

localhost-5000-admin-events- iphone 6 plus

localhost-5000-admin-events- iphone 6 plus 1

**Comment by sainiak009** [comment]
Above screenshots are for Iphone 6 plus screen

File: app/templates/gentelella/admin/super_admin/events/events.html [hunk 1]
```diff
@@ -305,13 +305,18 @@
     <script src="{{ url_for('static', filename='admin/lib/bootstrap-progressbar/bootstrap-progressbar.min.js') }}"></script>
     <script type="text/javascript">
         var table = $('.table.no-global-dt').DataTable({
-            "dom": '<"row"<"toolbar col-md-7"><"col-md-2"l><"col-md-3 pull-right"f>>tip',
+            "dom": '<"row"<"toolbar col-md-7"><"col-md-2"l><"col-md-3 pull-right"f>><".table-responsive"t>ip',
```
Comment (niranjan94) [inline app/templates/gentelella/admin/super_admin/events/events.html]: According to dataTables `dom` syntax, class names should be specified without a `.` (dot). 

So, it should be 
`<"row"<"toolbar col-md-7"><"col-md-2"l><"col-md-3 pull-right"f>><"table-responsive"t>ip`

**Comment by sainiak009** [comment]
@niranjan94 : Corrected the syntax for dom option of DataTables.

File: app/templates/gentelella/admin/super_admin/messages/messages.html [hunk 2]
```diff
@@ -94,12 +94,20 @@
...
+            "initComplete": function(settings, json){
+                table_width_adjust();
+                window.onresize = table_width_adjust;
+            },
             "pageLength": 25,
             "aaSorting": []
         });
         $('#notifs_table').DataTable({
-            "dom": '<"row"<"toolbar col-md-7"><"col-md-2"l><"col-md-3 pull-right"f>>tip',
+            "dom": '<"row"<"toolbar col-md-7"><"col-md-2"l><"col-md-3 pull-right"f>><".table-responsive"t>ip',
```
Comment (niranjan94) [inline app/templates/gentelella/admin/super_admin/messages/messages.html]: Please correct the dom syntax here too

**Pull request merged**